GitUI/CommandsDialogs/FormAddToGitIgnore.cs

```diff
@@ -70,25 +70,23 @@ private void AddToIgnoreClick(object sender, EventArgs e)
 
             try
             {
-                var fileName = ExcludeFile;
-                Validates.NotNull(fileName);
-                FileInfoExtensions.MakeFileTemporaryWritable(fileName, x =>
+                FileInfoExtensions.MakeFileTemporaryWritable(Validates.NotNull(ExcludeFile), fileName =>
                 {
                     StringBuilder gitIgnoreFileAddition = new();
```

Comment:
Just a thought. Why do we need to build the finished multi line but doubtful more than a few lines of ignores?  Could looping patterns and calling Write with each pattern and a new line be an option?